src/app/pcasts/controllers/merge_account_controller.py

```diff
@@ -0,0 +1,29 @@
+from app.pcasts.utils import facebook_utils, google_utils
+from . import *
+
+class MergeAccountController(AppDevController):
+
+  def get_path(self):
+    return '/users/merge/'
+
+  def get_methods(self):
+    return ['POST']
+
+  @authorize
+  def content(self, **kwargs):
+    user = kwargs.get('user')
+    token = request.args['access_token']
+    new_platform = request.args['platform']
+
+    if new_platform == "facebook":
```

Comment:
More nitpicking, same suggestions here for moving stuff to constants as down below

Is there a particular reason for why id_source for facebook is "fb" whereas here we are expecting "facebook"? Just want to keep it consistent when possible